src/OrchardCore.Modules/OrchardCore.ContentTypes/Services/ContentDefinitionService.cs

```diff
@@ -19,13 +19,15 @@ public class ContentDefinitionService : IContentDefinitionService
 
     private readonly IContentDefinitionManager _contentDefinitionManager;
     private readonly IEnumerable<IContentDefinitionEventHandler> _contentDefinitionEventHandlers;
+    private readonly IEnumerable<IContentDefinitionHandler> _contentDefinitionHandlers;
```

Comment:
There should only be triggered from the manager class not the service.  Instead resolve these in ContentDefinitionManager.cs class and invoke them only from the manager